packages/react-instantsearch-core/src/connectors/__tests__/connectInfiniteHits.js

```diff
@@ -51,21 +47,21 @@ describe('connectInfiniteHits', () => {
     });
 
     it('accumulate hits internally while changing hitsPerPage configuration', () => {
-      const context = createSingleIndexContext();
-      const getProvidedProps = connect.getProvidedProps.bind(context);
```

Comment:
We can keep this line, at least we bind it once for the test case.

Reply:
that might be inconsistent, since we no longer bind in the other tests

Reply:
Yes but you can do it for the full test suite.